src/community/utrecht/alert.tokens.json

```diff
@@ -0,0 +1,78 @@
+{
+  "utrecht": {
+    "alert": {
+      "icon": {
+        "gap": {
+          "value": "16px"
+        },
+        "error": {
+          "color": {
+            "value": "var(--of-color-danger)"
+          }
+        },
+        "info": {
+          "color": {
+            "value": "var(--of-color-info)"
+          }
+        },
+        "warning": {
+          "color": {
+            "value": "var(--of-color-warning)"
+          }
+        },
+        "ok": {
+          "color": {
+            "value": "var(--of-color-success)"
+          }
+        }
+      },
+      "padding": {
+        "block": {
+          "end": {
```

Comment:
the upstream design tokens are not split out that much:

https://nl-design-system.github.io/utrecht/storybook/?path=/story/css_css-alert--design-tokens

so this should be `"padding-block-end": {"value": "20px"},` instead